Packages/Sources/RootView/Home.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  SwiftUIView.swift
+//
+//
+//  Created by HH on 09/01/2024.
+//
+
+import NodeCryptoCore
+import SwiftUI
+
+struct Home: View {
+  @State private var activeTab: Tab = .search
+  @Namespace var animation
+
+  var body: some View {
+    VStack(spacing: 0) {
+      TabView(selection: $activeTab) {
+        Color.red
+          .tag(Tab.home)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Services")
+          .tag(Tab.search)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Notifications")
+          .tag(Tab.notifications)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Profile")
+          .tag(Tab.profile)
+          .toolbar(.hidden, for: .tabBar)
+      }
+
+      CustomTabBar()
+    }
+  }
+
+  @ViewBuilder
+  func CustomTabBar(tint: Color = .blue, inactiveTint: Color = .blue) -> some View {
```

Comment:
Please see [Swift API Design guidelines](https://www.swift.org/documentation/api-design-guidelines/) - Conventions. 
![image](https://github.com/hilalalhakani/NodeCrypto/assets/1016974/113069fb-9196-4f42-827a-6d02f2190b47)

Reply:
Oops! I didn't notice it